plugins/base/src/test/kotlin/expect/AbstractExpectTest.kt

```diff
@@ -40,9 +40,9 @@ abstract class AbstractExpectTest(
             ).also { logger.info("git diff command: ${it.command().joinToString(" ")}") }
                 .also { it.redirectErrorStream() }.start()
 
-            assertTrue(gitCompare.waitFor(gitTimeout, TimeUnit.MILLISECONDS)) { "Git timed out after $gitTimeout" }
+            assertTrue(gitCompare.waitFor(gitTimeout, TimeUnit.MILLISECONDS), "Git timed out after $gitTimeout")
             gitCompare.inputStream.bufferedReader().lines().forEach { logger.info(it) }
-            assertTrue(gitCompare.exitValue() == 0) { "${path.fileName}: outputs don't match" }
+            assertTrue(gitCompare.exitValue() == 0, "${path.fileName}: outputs don't match")
```

Comment:
I've fixed the specific places that you outlined since you reviewed them already, but I think such changes fall outside of the scope of this PR, so I won't look for any other similar improvement opportunities on my own

I'm all for correcting and improving all of the asserts, like using `assertContains` instead of `assertTrue(list.contains())`, but I think it's best done as a separate change that would be easier to review and will not be blocking anything